src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,261 @@
+## How to use Notification Triggers {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Notification Triggers API locally, without an origin trial token, enable the
+`#enable-experimental-web-platform-features` flag in `chrome://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chrome ⚠️ToDo, the Notification Triggers API will be available as an origin trial.
+Origin trials allow you to try new features and give feedback on their usability, practicality, and
+effectiveness, both to us, and to the web standards community. For more information, see the
+[Origin Trials Guide for Web Developers](https://googlechrome.github.io/OriginTrials/developer-guide.html).
+
+Request a [token](https://developers.chrome.com/origintrials/) for your origin. Add the token to
+your pages, there are two ways to provide this token on any pages in your origin: Add an
+`origin-trial` `<meta>` tag to the `head` of any page. For example, this may look something like:
+`<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">` If you can configure your server, you
+can also provide the token on pages using an `Origin-Trial` HTTP header. The resulting response
+header should look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+### Feature detection
+
+You can find out if the browser supports Notification Triggers by checking for the existence of the
+`showTrigger` property.
+
+```js
+if ("showTrigger" in Notification.prototype) {
+  /* Notification Triggers supported */
+}
+```
+
+### Scheduling a notification
+
+Scheduling a notification is similar to showing a regular push notification, with the difference
+being that you need to pass a `showTrigger` condition property with a `TimestampTrigger` object as
+the value to the notification's options object.
+
+```js
+const createScheduledNotification = async (tag, title, timestamp) => {
+  const registration = await navigator.serviceWorker.getRegistration();
+  registration.showNotification(title, {
+    tag: tag,
+    body: "This notification was scheduled 30 seconds ago",
+    showTrigger: new TimestampTrigger(timestamp + 30 * 1000)
+  });
+};
+```
+
+{% Aside %}
+  On desktop, Notifications Triggers will fire only if Chrome is running. On Android,
+  Notifications Triggers will fire completely independent from Chrome.
+{% endAside %}
+
+### Canceling a previously scheduled notification
+
+To cancel previously scheduled notifications first request a list of all notifications that match a
+certain tag through `ServiceWorkerRegistration.getNotifications()`. Note that you need to pass the
+`includeTriggered` flag for scheduled notifications to be included in the list.
+
+```js
+const tag = 'meeting';
+const cancelScheduledNotification = async (tag) => {
+  const registration = await navigator.serviceWorker.getRegistration();
+  const notifications = await registration.getNotifications({
+    tag: tag,
+    includeTriggered: true,
+  });
+
+  if (Array.isArray(notifications)) {
```

Comment:
Why do we need this check here? Doesn't `getNotifications()` always return an array?

Reply:
It's purely used as feature detection here.

Reply:
I'm not sure I understand. Why do feature detection here?

Reply:
In theory getting a list of scheduled notifications could be implemented, but scheduling notifications couldn't. So checking both.

Reply:
Actually, since it's just of theoretical nature, https://github.com/GoogleChrome/web.dev/pull/1744/files#diff-f5600abbbd2e7a1fd322c4c6b9092604 removes it.